src/Agent/NewRelic/Agent/Core/Attributes/AttributeDefinitionService.cs

```diff
@@ -289,6 +289,7 @@ private AttributeDefinition<object, object> CreateCloudSdkAttribute(string attri
                 .Create<object, object>(attribName, AttributeClassification.AgentAttributes)
                 .AppliesTo(AttributeDestinations.TransactionTrace)
                 .AppliesTo(AttributeDestinations.SpanEvent)
+                .WithConvert(x => x)
```

Comment:
What does this change enable? Or does it remove some behavior that we don't want it to have?

Reply:
If there's no conversion function defined, some attribute code paths fail silently (like `TrySetValue`). I'm not sure what the reasoning for that is, though. It's something should probably discuss.